scripts/import_from_application.py

```diff
@@ -1,11 +1,16 @@
 #!/usr/bin/env python3
 import argparse
+import sys
 
 import requests
-from app import app
-from config import Config
-from db.queries import bulk_insert_application_record
-from fsd_utils import CommonConfig
+
+sys.path.insert(1, ".")
```

Comment:
Could we try using `python -m` flag with path to the script as per https://github.com/communitiesuk/funding-service-design-assessment-store/blob/066dec1ea4003a01a2c15d17f86884959f71e88d/scripts/README.md?plain=1#L6 to try and avoid this workaround?

Reply:
Yeah that works 👍